python/desc/monitor/truth.py

```diff
@@ -0,0 +1,305 @@
+        return parameters of the objects with the desired columns as a
+        `pandas.DataFrame`
+
+        Parameters
+        ----------
+        idValue: int, optional, defaults to None
+            indexes of the astrophysical objects whose parameters are to be
+            obtained. If idValue is None, then all astrophysical objects with
+            ids in `self.idSequence` are used. If `self.idSequence` is None,
+            then all astrophysical objects are used.
+        """
+        df = pd.read_sql_query(self.buildquery(idValue=idValue),
+                               self.dbConnection,
+                               index_col=self.idCol,
+                               coerce_float=False)
+        return df
+
+    def astro_object(self, idValue, mjdOffset=59580.):
+        """
+        instance of the catsim representation of the astrophysical object.
+
+        Parameters
+        ----------
+        idValue: int, mandatory
+            index of the astro_object
+        mjdOffset: float, optional, defaults to 59580.
+            offset in time parameters in the database for the transient objects
+
+        Returns
+        -------
+        Instance of astro_object with parameters from the database
+
+        Examples
+        --------
+        >>> sn = reflc.astro_object(idValue=6001163623700)
+
+        """
+        df = self.get_params(idValue)
+        sn = SNObject(ra=df.snra.values[0], dec=df.sndec.values[0])
+        paramDict = dict()
+        for param in ['t0', 'x0', 'x1', 'c']:
+            paramDict[param] = df[param].values
+        paramDict['t0'] += mjdOffset
+        paramDict['z'] = df.redshift.values[0]
+        sn.set(**paramDict)
+        return sn
+
+    def lightCurve(self, idValue, observations, bandpassDict,
+                   format='dataframe',
+                   keys=['expMJD', 'filter', 'fiveSigmaDepth'],
+                   photParams=None):
+        """
+        return the light curve of the object for observations as a
+        `pandas.Dataframe`.
+        Parameters
+        ----------
+        idValue: integer, mandatory
+            index of astrophysical object
+        observations: mandatory, allowed formats decided by format variable
+            observations corresponding to which the light curve is obtained
+        format: string, optional, defaults to dataframe
+            format in which observations are available
+        keys: aliases for columns time (in MJD), bandpass Name, fivesigmaDepth
+            (mags)
+        photParams: instance of `sims.photUtils.PhotParams`, optional,
+            defaults to None
+            Describes the observing conditions and telescope for the
+            observation. The default value of None instantiates this for the
+            LSST site and telescope.
+
+        Returns
+        -------
+        astropy.Table containing the following columns at the minimum
+        ['time', 'flux', 'fluxerr', 'band']. If a set of indexes are provided
```

Comment:
It would be good to state here that the flux is in maggies and the time is in MJD.